app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -2,104 +2,102 @@
   background: govuk-colour("white");
   margin-top: govuk-spacing(6);
 
-  @include govuk-media-query($from: tablet) {
+  @include govuk-media-query($from: desktop) {
     margin-top: govuk-spacing(9);
-    border-bottom: 1px solid govuk-colour("white");
   }
 
+  // New design has the box flush with edges of smaller screens
+  // We need to compensate for `govuk-width-container` margins:
+  @include govuk-media-query($until: tablet) {
+    margin-right: -15px;
+    margin-left: -15px;
+  }
+  @include govuk-media-query($from: tablet, $until: desktop) {
+    margin-right: -30px;
```

Comment:
Yes, these margins are to counteract the outer margins of the top level wrapper `govuk-width-container`. This is because the new design runs flush to the edges of the screen. The preview app won't render this correctly, but it should render correctly when viewed through a frontend app. Is that not the case?

Reply:
For reference, this screenshot is from being rendered inside static. I chatted to Matt about this offline and will catch up on it in a bit.

Reply:
OK, there were a number of different issues causing this. I've now changed the way that the negative margins and inner padding on elements interact. Now, elements won't hit the edge of the screen, and their inner elements will align with other elements in the content above.

Reply:
I've re-opened this comment as the desktop margins now need to change after a design review. Once those are done, we should check that the mobile margins also match what's in Figma.